--- /Users/dfrank/.config/homebrew/dump/Brewfile.2025-03-03T08:59:18-0800.MLF67N6G9N5W.preinstall	2025-03-03 08:59:23
+++ /Users/dfrank/.config/homebrew/dump/Brewfile.2025-03-03T08:59:18-0800.MLF67N6G9N5W.postinstall	2025-03-03 08:59:42
@@ -28,2 +27,0 @@
-# Official Amazon AWS command-line interface
-brew "awscli"
@@ -86,2 +83,0 @@
-# Fast linters runner for Go
-brew "golangci-lint"
@@ -90,2 +85,0 @@
-# Curl for GraphQL with autocomplete, subscriptions and GraphiQL
-brew "graphqurl"
@@ -96,4 +89,0 @@
-# Development kit for the Java programming language
-brew "openjdk"
-# Java-based scripting language
-brew "groovy"
@@ -120,2 +109,0 @@
-# Easy way to access the system keyring service from python
-brew "keyring"
@@ -126,2 +113,0 @@
-# XML security library
-brew "libxmlsec1"
@@ -162,2 +147,0 @@
-# Cloud native development platform
-brew "pulumi"
@@ -168,2 +151,0 @@
-# Interpreted, interactive, object-oriented programming language
-brew "python@3.11"
@@ -190,2 +171,0 @@
-# Editor of encrypted files
-brew "sops"
@@ -220,2 +199,0 @@
-# Vulnerability scanner for container images, file systems, and Git repos
-brew "trivy"
@@ -226,2 +203,0 @@
-# Secures, stores, and tightly controls access to secrets
-brew "vault"
@@ -230,2 +205,0 @@
-# Tool for creating isolated virtual python environments
-brew "virtualenv"
