--- /Users/dfrank/.config/homebrew/dump/Brewfile.2025-03-03T09:05:48-0800.MLF67N6G9N5W.preinstall	2025-03-03 09:05:54
+++ /Users/dfrank/.config/homebrew/dump/Brewfile.2025-03-03T09:05:48-0800.MLF67N6G9N5W.postinstall	2025-03-03 09:06:53
@@ -27,0 +28,2 @@
+# Official Amazon AWS command-line interface
+brew "awscli"
@@ -83,0 +86,2 @@
+# Fast linters runner for Go
+brew "golangci-lint"
@@ -85,0 +90,2 @@
+# Curl for GraphQL with autocomplete, subscriptions and GraphiQL
+brew "graphqurl"
@@ -89,0 +96,4 @@
+# Development kit for the Java programming language
+brew "openjdk"
+# Java-based scripting language
+brew "groovy"
@@ -109,0 +120,2 @@
+# Easy way to access the system keyring service from python
+brew "keyring"
@@ -113,0 +126,2 @@
+# XML security library
+brew "libxmlsec1"
@@ -147,0 +162,2 @@
+# Cloud native development platform
+brew "pulumi"
@@ -171,0 +188,2 @@
+# Editor of encrypted files
+brew "sops"
