--- /Users/dmf/.config/homebrew/dump/Brewfile.2025-03-29T15:23:40-0700.dmf-macbookpro2023.local.preinstall	2025-03-29 15:23:49
+++ /Users/dmf/.config/homebrew/dump/Brewfile.2025-03-29T15:23:40-0700.dmf-macbookpro2023.local.postinstall	2025-03-29 15:30:07
@@ -34,2 +33,0 @@
-# Interpreted, interactive, object-oriented programming language
-brew "python@3.12"
@@ -85,0 +84,2 @@
+# Interpreted, interactive, object-oriented programming language
+brew "python@3.12"
@@ -171,0 +172,2 @@
+# CLI and dbus interface for WhisperSystems/libsignal-service-java
+brew "signal-cli"
